presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraServer.java

```diff
@@ -59,10 +59,16 @@
 
     public CassandraServer()
             throws Exception
+    {
+        this("cassandra:2.2");
```

Comment:
Extract `cassandra:` and pass only version `2.2`. Then, rename subsequent `dockerImageName` to `cassandraVersion`.